pycbc/io/hdf.py

```diff
@@ -16,7 +16,9 @@
 from pycbc_glue.ligolw import utils as ligolw_utils
 from pycbc_glue.ligolw.utils import process as ligolw_process
 
+from pycbc import types
```

Comment:
Perhaps its most closely related to ``coinc`` but the function is a bit broader than coinc triggers. Could add a ``pycbc/events/triggers.py`` and put it in there and move the background/bank bin CLI function to there as well. Unless you have another suggestion.